tests/schemas/views.py

```diff
@@ -119,8 +119,11 @@ class ExampleValidatedSerializer(serializers.Serializer):
             MinLengthValidator(limit_value=2),
         )
     )
-    decimal1 = serializers.DecimalField(max_digits=6, decimal_places=2)
-    decimal2 = serializers.DecimalField(max_digits=5, decimal_places=0,
+    decimal1 = serializers.DecimalField(max_digits=6, decimal_places=2, coerce_to_string=False)
```

Comment:
Let's have some test cases where `coerce_to_string ` is not set.

Reply:
Done.